src/os_common/os_fhandle.c

```diff
@@ -299,9 +299,9 @@ __handle_close(WT_SESSION_IMPL *session, WT_FH *fh, bool locked)
 	conn = S2C(session);
 
 	if (fh->ref != 0) {
-		__wt_errx(session,
-		    "Closing a file handle with open references: %s", fh->name);
 		WT_TRET(EBUSY);
+		__wt_err(session, ret,
```

Comment:
This change is not required. 'ret' will still have '0'.

Reply:
5e0c171

As discussed, that line wasn't doing anything so I've removed it and changed the call back to `wt_errx`.